dumps/git-auth.yaml

```diff
@@ -0,0 +1,42 @@
+service: !u FP.Service.Authentication
```

Comment:
What does `!u` do?

Reply:
`!u` is a YAML custom data type marker. In this case it is expanded by `convert-dumps.js` into a UUID pulled either from `@amrc-factoryplus/utilities` or from `lib/uuids.js` as appropriate.

I'm trying to get to the point of having a SSoT for these UUIDs. It is concerning me that we are copy-pasting them all the time.